src/Authentication/Controllers/SystemRegisterController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.Platform.Authentication.Core.Models;
+using Altinn.Platform.Authentication.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Authentication.Controllers;
+
+/// <summary>
+/// CRUD API for SystemRegister
+/// </summary>
+[Route("authfront/api/v1/systemregister")]
```

Comment:
We can also discuss about the route organization either in grooming or refinement this week

Reply:
Ok, we need to add Role to the Authorize, but the route remains the same, re discussion in Refinement meeting.